src/main/kotlin/no/nav/familie/oppdrag/simulering/SimulerBeregningRequestMapper.kt

```diff
@@ -0,0 +1,93 @@
+package no.nav.familie.oppdrag.simulering
+
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsoppdrag
+import no.nav.familie.kontrakter.felles.oppdrag.Utbetalingsperiode
+import no.nav.familie.oppdrag.iverksetting.*
+import no.nav.system.os.entiteter.typer.simpletypes.FradragTillegg
+import no.nav.system.os.entiteter.typer.simpletypes.KodeStatusLinje
+import no.nav.system.os.tjenester.simulerfpservice.simulerfpserviceservicetypes.Oppdrag
+import no.nav.system.os.tjenester.simulerfpservice.simulerfpserviceservicetypes.Oppdragslinje
+import org.springframework.stereotype.Component
+import java.time.format.DateTimeFormatter
+
+@Component
+class SimulerBeregningRequestMapper {
+    private val fpServiceGrensesnittFactory =
+            no.nav.system.os.tjenester.simulerfpservice.simulerfpservicegrensesnitt.ObjectFactory()
+    private val fpServiceTypesFactory = no.nav.system.os.tjenester.simulerfpservice.simulerfpserviceservicetypes.ObjectFactory()
+    private val oppdragSkjemaFactory = no.nav.system.os.entiteter.oppdragskjema.ObjectFactory()
+
+    private val tidspunktFormatter = DateTimeFormatter.ofPattern("yyyy-MM-dd-HH.mm.ss.SSSSSS")
+
+    fun tilSimulerBeregningRequest(utbetalingsoppdrag: Utbetalingsoppdrag): no.nav.system.os.tjenester.simulerfpservice.simulerfpservicegrensesnitt.SimulerBeregningRequest =
```

Comment:
Går det an å gjøre noe med disse import'sene slik at det ikke blir så vanskelig å lese?

Reply:
Dessverre det blir litt vanskelig når de har samme klassenavn på to forskellige klasser.